src/query-builder/SelectQueryBuilder.ts

```diff
@@ -3740,8 +3755,18 @@ export class SelectQueryBuilder<Entity extends ObjectLiteral>
 
     /**
      * Loads raw results from the database.
+     * @param timestamp get datasets at a specific timestamp (temporal tables)
+     * if undefined use date in the future to get the current datasets
      */
-    protected async loadRawResults(queryRunner: QueryRunner) {
+    protected async loadRawResults(
+        queryRunner: QueryRunner,
+        timestamp = addYears(new Date(), 10),
```

Comment:
Before timestamp was an optional parameter, by adding a default value you make mandatory, and it will be set for everyone. Not sure if this is the right approach.

Reply:
Thanks for the review. Default parameters are optional. This means this method is still backward compatible.